build.cmd

```diff
@@ -0,0 +1,5 @@
+@echo off
```

Comment:
For Another Review: we could add a script to pack

Reply:
I didn't get the point here, could you please give me more information?

Reply:
It would be nice to get a script file that would be called pack.cmd and pack.sh which would do "fake pack". It can be painful under linux systems to call mono with the fax and so on

Reply:
Fake pack is done by **Pack** target inside **build.fsx** file that will be launched by calling **build.cmd Pack** or **build.sh Pack**, why do we need another script file?

Reply:
ah my bad, I didn't get it right!